shared/altair-devnet-0/altair-devnet-0.yaml

```diff
@@ -0,0 +1,73 @@
+# Altair interop devnet config
```

Comment:
Could we please rename this file to `config.yaml`?

I suspect most implementations ascribe meaning to the filename, as previously there were multiple files `altair.yaml`, `phase0.yaml`, etc.

Reply:
I've made the change and renamed it to `config.yaml`